### PR TITLE
6 sign up

### PR DESCRIPTION
**Pull request by Leigan0**
Login in feature complete 
User can log in 
Data validated against database
Errors raised and displayed if incorrect information given
Errors raised for incorrect login information

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -3,11 +3,21 @@
 ENV['RACK_ENV'] ||= 'development'
 
 require 'sinatra/base'
+require 'sinatra/flash'
 require './database_setup.rb'
 
 class DwellBNB < Sinatra::Base
 
   enable :sessions
+  set :session_secret, '7a929c0c04b166475878f813dd25d869b5ad4e631eed3194edb2555e22039a94'
```
Comment (alextwilson) [inline app/app.rb:13]: Could we not hide this secret as an ENV variable?

**Comment by lewmoore** [comment]
:shipit:

File: app/app.rb (hunk 1 above)
Comment (alextwilson) [reply app/app.rb:13]: But otherwise :shipit:

Comment (Leigan0) [reply app/app.rb:13]: At some point I think we could

**Comment by Gleoman** [comment]
:shipit:

**Comment by lewmoore** [comment]
:shipit:

**Pull request merged**